test/signin.js

```diff
@@ -0,0 +1,30 @@
+let app = require('../app');
+let chai = require('chai');
+let request = require('supertest')
+let should = chai.should();
+request = request('http://localhost:3000')
```

Comment:
Yes. request(app).post(..) worked!  thanks for catching that. Good to know that reassigning variables is not good practice. I will remember that.